howitzer.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency 'sexy_settings'
 
   gem.add_development_dependency('aruba')
-  gem.add_development_dependency('cucumber', '~>2')
+  gem.add_development_dependency('cucumber')
```

Comment:
We could remove the line, because Cucumber will be automatically installed with Aruba. I added it specially to limit with Cucumber 2. So, please remove